src/orderbook.js

```diff
@@ -106,7 +140,16 @@ function OrderBook(remote,
       switch (action) {
         case 'add':
           if (++self._listeners === 1) {
+            self._shouldSubscribe = true;
             self.subscribe();
+
+            self._remote.on('transaction', updateFundedAmountsWrapper);
+            self._remote.once('disconnect', onDisconnect);
+
+            if (self._isAutobridgeable) {
```

Comment:
I specifically marked subscribe/unsubscribe methods as private in comments - anyway there is no point to subscribe without adding listeners, and anyway they not work right if called from outside.